src/RemixDevTools/index.ts

```diff
@@ -1,3 +1,12 @@
 export { RemixDevTools } from "./RemixDevTools";
 export { useRemixForgeSocketExternal as useRemixForgeSocket } from "./hooks/useRemixForgeSocket";
 export { initClient, initServer } from "./init/project";
+
+export interface Tab {
+  name: string;
+  icon: JSX.Element;
+  id: string;
+  component: JSX.Element;
+  requiresForge: boolean;
+  hideTimeline: boolean;
+}
```

Comment:
I don't mind the fix but I think the root index file should only export the types, would this work if you did:
```jsx import type { Tab } from ".." export type { Tab }```
or even better:
```jsx import type { Tab } from ".." export type { Tab as RemixDevToolsPluginReturn }```

Reply:
also this is indicative of the fact that you might be running tsc on your node_modules which you shouldn't do, you should for sure add it to the ignored files in your tsconfig or add the skipLibCheck option

Reply:
I'll check your solution.

In general, I agree about ignoring `node_modules` for typecheck. I'm using prisma which [generates types to node_modules](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-prismaclient/generating-prisma-client) by default. It allows consumers to specify generated output, but that requires more work to consume Remix-Dev-Tools and some people are not gona ignore typecheck on `node_modules `

Reply:
I've updated the change, I think it's what you were asking for.

Reply:
hey @AlemTuzlak , are you cool w/this or do i need to change something?

Reply:
> hey @AlemTuzlak , are you cool w/this or do i need to change something?

I am so sorry I missed the update you made just saw the notification. Will check asap

Reply:
> I've updated the change, I think it's what you were asking for.

Mind checking if the latest release solved it?

Reply:
hey @AlemTuzlak,
i updated to v2.1.1, getting a similar error cuz you're importing  `Tab` from a directory that the `dist/` doesn't have. 
I've added a new commit that fixes it.